tools/llm_bench/task/speech_to_text_generation.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import os
+import time
+import datetime
+import numpy as np
+from pathlib import Path
+import hashlib
+import logging as log
+import llm_bench_utils
+import llm_bench_utils.model_utils as model_utils
+import llm_bench_utils.metrics_print as metrics_print
+import llm_bench_utils.gen_output_data as gen_output_data
+import llm_bench_utils.parse_json_data as parse_json_data
+
+FW_UTILS = {'pt': llm_bench_utils.pt_utils, 'ov': llm_bench_utils.ov_utils}
+
+
+def run_speech_2_txt_generation(raw_speech, pipe, args, num, md5_list, speech_id,
+                                iter_data_list, mem_consumption, processor):
+    result_md5_list = []
+    max_rss_mem_consumption = ''
+    max_uss_mem_consumption = ''
+    max_shared_mem_consumption = ''
+    if (args['mem_consumption'] == 1 and num == 0) or args['mem_consumption'] == 2:
+        mem_consumption.start_collect_memory_consumption()
+    start = time.perf_counter()
+    result_text = pipe.generate(
+        raw_speech,
+        max_new_tokens=1000,
+        # 'task' and 'language' parameters are supported for multilingual models only
+        language="<|en|>",
+        task="transcribe",
+        return_timestamps=True
```

Comment:
@as-suvorov, I remember discussion that we want to measure perf without timesteps, but it required to have restriction on audio duration. Should we make it configurable for benchmarking or measure only with/without timesteps

Reply:
Yes, good point. Transcription with enabled timestamps is expected to be a bit slower. It will be great if we can compare both modes. I think we can have `--long-form/--short-form`, `--return_timestamps` or similar flags.

As for perf metrics. For the moment time per output token and throughput can be reported higher than actual with timestamps enabled. I experienced this behavior with distil models only so far. I think we can enable benchmark with timestamps already and I'll work on perf_metrics correctness.